res/controllers/midi-components-0.0.js

```diff
@@ -1266,7 +1348,7 @@
             this.effectFocusButton.trigger();
 
             this.enableOnChannelButtons.forEachComponent(function(button) {
-                if (button.midi !== undefined) {
+                if (button.midiOut !== undefined) {
```

Comment:
```suggestion
                if (button.midiOut.length > 0) {
```

`midiOut` is never `undefined` anymore.